### PR TITLE
Create works section

### PR DESCRIPTION
**Pull request by abenaTenkorang**
in this branch  I built the middle section of the mobile website.

### CODE REVIEW

**Comment by abenaTenkorang** [comment]
final update was made on the create work section branch

File: index.html [hunk 1]
```diff
@@ -66,7 +66,258 @@ <h1 class="header-title ">I'm Abigail T Glad to see you!
...
+              </ul>
+
+              <a class="card-link" href="#">See Project</a>
+              
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+</section>
```
Comment (Kamba56) [inline index.html]: - [x] Please note that all your cards should be in one section. Your cards can be articles in a section or just divs in a section. Also, you are to align the contents of the section (that is the cards) using CSS grid as demanded by project requirements. Kindly fix this, please.

File: index.html [hunk 2]
```diff
@@ -66,7 +66,258 @@ <h1 class="header-title ">I'm Abigail T Glad to see you!
...
+
+<!-- top section -->
+<section class="top-section">
+      
+  <h1 class="top-title "> About Myself</h1>
```
Comment (Kamba56) [inline index.html]: - [ ] Please note that you should have only one `h1` in the whole of your document, kindly change this to any other heading tag(from `h2` to `h5`). This is SEO best practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.

File: index.html [hunk 3]
```diff
@@ -66,7 +66,258 @@ <h1 class="header-title ">I'm Abigail T Glad to see you!
...
+    <h2 class="techies-title">Frameworks</h2>
+    <img class="forward-arrow" src="./images/arrow.png" alt="Arrow Forward">
+  </div>
+  <hr class="hr">
+
+  <!-- Skills -->
+  <div class="techies">
+    <h2 class="skills-title">Skills</h2>
+    <img class="techies-forward-arrow" src="./images/arrow.png" alt="Arrow Forward">
+  </div>
```
Comment (Kamba56) [inline index.html]: - [ ] Kindly consider changing your list of technologies to a nested list as demanded by the project requirements. A nested list looks like this;
```html
<ul>
  <li>
    <h2>Languages</h2>
    <ul>
      <li>CSS</li>
      ...
    </ul>
  </li>
  <li>
    <h2>Frameworks</h2>
  </li>

  <li>
    <h2>Skills</h2>
  </li>
</ul>
```

File: style.css [hunk 4]
```diff
@@ -98,3 +101,343 @@ body {
   width: 20px;
   height: 20px;
 }
+
+@media screen and (max-width: 768px) {
```
Comment (Kamba56) [inline style.css]: - [x] Please remove media queries for now. You are expected to build this page mobile-first, which means all your mobile styling will be in the CSS file before you later write desktop styling in media queries. So just remove the media queries and move the styling in them outside the queries.

File: style.css [hunk 5]
```diff
@@ -98,3 +101,343 @@ body {
...
+  .btn-link {
+    align-items: center;
+    text-decoration: none;
+    padding: 12px;
+    gap: 10px;
+    width: 124px;
+    height: 48px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+  }
```
Comment (Kamba56) [inline style.css]: - [ ] Consider adding a hover and pressed state to your buttons to match the figma design. You can use the `:hover` and `:active` pseudo-classes.
btn1

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @abenaTenkorang, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great UI design
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (abenaTenkorang) [reply index.html]: issue resolved. Thank you

File: style.css (hunk 4 above)
Comment (abenaTenkorang) [reply style.css]: Issue resolved. Thank you

File: style.css (hunk 5 above)
Comment (abenaTenkorang) [reply style.css]: issue resolved. Thank you

File: index.html (hunk 3 above)
Comment (abenaTenkorang) [reply index.html]: Issue resolved. Thanks for the feedback @Kamba56

File: index.html (hunk 1 above)
Comment (abenaTenkorang) [reply index.html]: resolved issue 
thank you

**Comment by abenaTenkorang** [comment]
all issues raised have been resolve

**Review by Debas-31 — APPROVED** [review]
> ## Hi @abenaTenkorang  :wave:, 

# Status: Approved :green_circle: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Linters setup :heavy_check_mark:
- GitHub flow :heavy_check_mark: 
- Project working :heavy_check_mark: 
- UI design :heavy_check_mark: 
- Figma template usage :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._d

**Pull request merged**